src/string_bytes.cc

```diff
@@ -392,15 +392,6 @@ size_t StringBytes::Write(Isolate* isolate,
 }
 
 
-bool StringBytes::IsValidString(Local<String> string,
-                                enum encoding enc) {
-  if (enc == HEX && string->Length() % 2 != 0)
-    return false;
-  // TODO(bnoordhuis) Add BASE64 check?
```

Comment:
Can you move this to the JS land function? cc @bnoordhuis

Reply:
Fixed.